vite.config.js

```diff
@@ -1,9 +1,10 @@
-import { defineConfig } from "vite";
 import vue from "@vitejs/plugin-vue";
 import path from "path";
+import { defineConfig } from "vite";
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  base: "./",
```

Comment:
vite will use relative paths, this is required cause absolute path in electron doesn't work.

**This does not affect the web build**